tests/sentry/snuba/test_subscriptions.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+
+from sentry.snuba.models import QueryAggregations, QueryDatasets, QuerySubscription
+from sentry.snuba.subscriptions import (
+    create_snuba_subscription,
+    delete_snuba_subscription,
+    update_snuba_subscription,
+)
+from sentry.testutils import TestCase
+
+
+class CreateSnubaSubscriptionTest(TestCase):
+    def test(self):
+        type = "something"
+        dataset = QueryDatasets.EVENTS
+        query = "level:error"
+        aggregation = QueryAggregations.TOTAL
+        time_window = 10
+        resolution = 1
+        subscription = create_snuba_subscription(
+            self.project, type, dataset, query, aggregation, time_window, resolution
+        )
+        assert subscription.project == self.project
+        assert subscription.type == type
+        assert subscription.subscription_id != ""
+        assert subscription.dataset == dataset.value
+        assert subscription.query == query
+        assert subscription.aggregation == aggregation.value
+        assert subscription.time_window == time_window
+        assert subscription.resolution == resolution
+
+
+class UpdateSnubaSubscriptionTest(TestCase):
+    def test(self):
+        subscription = create_snuba_subscription(
+            self.project,
+            "something",
+            QueryDatasets.EVENTS,
+            "level:error",
+            QueryAggregations.TOTAL,
+            10,
+            1,
+        )
+
+        query = "level:warning"
+        aggregation = QueryAggregations.UNIQUE_USERS
+        time_window = 20
+        resolution = 2
+        old_subscription_id = subscription.subscription_id
+        update_snuba_subscription(subscription, query, aggregation, time_window, resolution)
```

Comment:
Are there any failure scenarios that should have tests?

Reply:
Probably a couple. I'll fill these tests out more once I move things to tasks, since that will change the flow here a bunch.